test/ExecutionFramework.cpp

```diff
@@ -182,15 +182,15 @@ void ExecutionFramework::sendMessage(bytes const& _data, bool _isCreation, u256
 		message.kind = EVMC_CALL;
 		message.destination = EVMHost::convertToEVMC(m_contractAddress);
 	}
-	message.gas = m_gas.convert_to<int64_t>();
+	message.gas = InitialGas.convert_to<int64_t>();
```

Comment:
Constants should be either full caps or just regular variables. UpperCamelCase is reserved for types.

Reply:
So how about full caps? Can we start using that everywhere? The `m_` was especially misleading in this case but I often find myself wishing we had some convention for constants to easily distinguish them from normal variables.

Reply:
I think the coding style guide says `c_` for constants.

Reply:
Unfortunately it does not. Should that be preferred then? You mentioned that convention before but I don't remember seeing it actually used anywhere.